.github/workflows/compatibility.yaml

```diff
@@ -0,0 +1,126 @@
+name: Compatibility of
```

Comment:
More of a preference:
```suggestion
name: Tool Compatibility Test
```

Reply:
Generally agree but I used `Compatibility of` here because in the CI it'll show up as

`Compatibility of/latest` and
`Compatibility of/nightly`

Still want to change?

Reply:
I see, that works.